docs/Contributing/README.md

```diff
@@ -10,16 +10,16 @@ Look at Fleet's full test suite and integration tests.
 Learn about creating and updating database migrations.
 
 ### [Committing changes](./Committing-Changes.md) 
-Read to learn how to merge changes into the codebase.
+Learn how to merge changes into the codebase.
 
 ### [Releasing Fleet](./Releasing-Fleet.md) 
 Learn the process for how to release Fleet.
 
 ### [Seeding data](./Seeding-Data.md) 
-Read this guide on adding fake data to your development instance.
+Learn how to add fake data to your development instance.
 
 ### [API for contributors](./API-for-contributors.md) 
 Get to grips with Fleet API routes. This documentation is helpful for developing or contributing to Fleet.
 
 ### [FAQ](./FAQ.md) 
-Read for commonly asked questions and answers about contributing to Fleet as part of the Fleet community.
+Commonly asked questions and answers about contributing to Fleet as part of the Fleet community.
```

Comment:
```suggestion
Find commonly asked questions and answers about contributing to Fleet as part of our community.
```
Added a verb for consistency with the other lines and tried reducing the mentions of Fleet in the same sentence.

Reply:
Excellent. I thought what I'd done sounded a bit odd, so thanks for that, @chris-mcgillicuddy.